draft-ietf-ppm-dap.md

```diff
@@ -1440,21 +1470,17 @@ struct {
 } InputShareAad;
 ~~~
 
-The Leader responds to well-formed requests with HTTP status code 201
-Created. Malformed requests are handled as described in {{errors}}.
```

Comment:
We need to be more specific than this in describing how to handle malformed requests